source/declarations.tex

```diff
@@ -8793,7 +8793,7 @@
 Function \tcode{g}'s second parameter has a \tcode{carries_dependency} attribute,
 but its first parameter does not. Therefore, function \tcode{h}'s first call to
 \tcode{g} carries a dependency into \tcode{g}, but its second call does not. The
-implementation might need to insert a fence prior to the second call to
+implementation can be required to insert a fence prior to the second call to
```

Comment:
"can be required" sounds like the programmer can cause the implementation to be required to do this, which isn't really right. This solves that problem but I'm still not overjoyed by it:
```suggestion
implementation can need to insert a fence prior to the second call to
```

Reply:
"can need"?

ugh

"It is possible that the implementation needs to insert a fence..."